pkg/components/components_test.go

```diff
@@ -94,4 +94,19 @@ var _ = Describe("Components", func() {
 			Expect(ris[2].Ref).To(Equal("quay.io/kubevirt/ovs-cni-marker@sha256:0f08d6b1550a90c9f10221f2bb07709d1090e7c675ee1a711981bd429074d620"))
 		})
 	})
+
+	Context("When calculating ciphers", func() {
+		It("should not generate duplicates", func() {
```

Comment:
Where are the duplicated ciphers ?

Reply:
Here for example:
https://github.com/kubevirt/cluster-network-addons-operator/blob/3a2b93b4259772257ead6c28ba748936da28e0bf/manifests/cluster-network-addons/0.90.1/network-addons-config.crd.yaml#L1523-L1537

Reply:
What I mean is that we should mock those, we are not really enforcing duplicated on the test.

Reply:
The goal of this test(and this PR overall) is to ensure we don't emit duplicates in the "ciphers" enums, the fact that they do exist or not in the original slice is almost an implementation detail.

If you want to do mock testing on something this small, please go ahead, you can PR my branch, fix on top or do whatever you prefer.

Reply:
Ok